packages/core/src/context/ContextSystem.ts

```diff
@@ -44,7 +66,7 @@ export class ContextSystem extends System
         this.handleContextLost = this.handleContextLost.bind(this);
         this.handleContextRestored = this.handleContextRestored.bind(this);
 
-        renderer.view.addEventListener('webglcontextlost', this.handleContextLost, false);
+        (renderer.view as any).addEventListener('webglcontextlost', this.handleContextLost, false);
```

Comment:
I presume the issue here is that the view has an explicit list of events it emits - is it maybe possible to just modify the list of events nad callbacks it supports rather than doing this cast to `any`?

Reply:
`webglcontextlost` is not standard event, it is from extension.
There aren't serious reason for override events list for 1 event.